custom/icds_reports/ucr/data_sources/cbe_form.json

```diff
@@ -55,6 +55,24 @@
           "name": "user_location_id"
         }
       },
+      {
+        "display_name": "Supervisor ID",
+        "datatype": "string",
+        "expression": {
+          "type": "ancestor_location",
+          "location_id": {
+            "type": "named",
+            "name": "user_location_id"
+          },
+          "location_type": "supervisor",
+          "location_property": "_id"
+        },
+        "is_primary_key": false,
+        "transform": {},
+        "is_nullable": true,
+        "type": "expression",
+        "column_id": "supervisor_id"
```

Comment:
There are currently 45 forms in this datasource. I'll just rebulid the whole table instead of writing a query